### DISCUSSION TITLE
如何安装TVM？

### DISCUSSION THREAD

**Discussion by kiloGrand** [mlc-ai/mlc-zh#17] [Q&A]
**安装出错，怎么在window anaconda安装呀？**
```
❯ python -m  pip install mlc-ai-nightly -f https://mlc.ai/wheels
Looking in indexes: https://mirrors.aliyun.com/pypi/simple/
Looking in links: https://mlc.ai/wheels
ERROR: Could not find a version that satisfies the requirement mlc-ai-nightly (from versions: none)
ERROR: No matching distribution found for mlc-ai-nightly
```
可不可以直接下载<https://github.com/mlc-ai/utils/releases/download/v0.9.dev0/mlc_ai_nightly-0.9.dev1661%2Bg2b34ced6c-cp37-cp37m-win_amd64.whl>，然后安装

**Comment by Hzfengsy** [comment]
目前windows版本只build了python 3.7，如果使用anaconda可以讲python版本调到3.7再安装，或者使用WSL或者Linux系统。

**Reply by kiloGrand** [reply]
还真是python版本的问题，3.8的装不了，3.7的装上了

**Reply by Belonger** [reply]
可以吗，我的python也是3.7,但总是安装失败
![1](https://user-images.githubusercontent.com/57164274/176323292-d8c744a1-3817-48a9-aba9-87a4a24d0d95.png)
并且在后面加上 -i https://pypi.tuna.tsinghua.edu.cn/simple也不成功

**Reply by Belonger** [reply]
![1](https://user-images.githubusercontent.com/57164274/176323662-575b61ca-03a0-4cf7-86a2-823751889a61.png)
报错问题

**Reply by Belonger** [reply]
在jupyter notebook中安装
![1](https://user-images.githubusercontent.com/57164274/176323776-544122c8-9de1-43ef-9c6c-83844d42f020.png)
我现在的位置在中国浙江，并且已经搭了梯子，还是不可以

**Reply by Belonger** [reply]
安装其他的，比如numpy, pandas, pytorch是没有问题的

**Reply by junrushao** [reply]
这个看起来是没有连上网站诶...

![image](https://user-images.githubusercontent.com/22515877/176327708-18510a70-bc72-4145-93f8-c9d0f06dc275.png)

**Reply by Hzfengsy** [reply]
mlc的wheel并不是走的pypi官方，所以镜像网站是同步不到的。
至于windows的terminal走代理流量需要单独设置，并不是浏览器能翻命令行就能翻的

**Reply by effrey-liu** [reply]
你好，我的anaconda虚拟环境中的python版本是3.7.16，但仍然报错
![image](https://github.com/mlc-ai/mlc-zh/assets/64138454/09b5be9b-3e38-4437-808d-6f3244680000)

**Comment by junrushao** [comment]
个人的经验，windows 上比较推荐通过 anaconda 的 python 发行版本安装 tvm

**Comment by Hyaloid** [comment]
你把镜像换一个 换成豆瓣的试试？

发自我的iPhone

**Comment by yingluAMD** [comment]
参看ReadMe,把`eval_notebook`设置成`False`试试，这样就没TVM的要求了，个人本地尝试成功。
Change `eval_notebook = True` to `eval_notebook = False` in `config.ini`.
